### PR TITLE
UI update

### LINKED ISSUE #76: Spruce up UI for curriculum app
(merged two stories to create this one)

---

From [ch3847](https://app.clubhouse.io/learnersguild/story/3847)

Some simple design tweaks we could do:

- Use Material UI
- Convert to Guild colors, typefaces
- Use style idioms for buttons, layout, etc.

---

From [ch3403](https://app.clubhouse.io/learnersguild/story/3403)

### colors

We need to ensure that the branding (colors, mostly) align with the Guild branding guide.

### favicon

We should always use the Learners Guild icon for favicons for all services. This was a Q1 goal of "internal brand matching external brand".

**Side-note:** Please don't create new iconography in any of our services!

#### Details

The "right" way to do it (that works on all devices in a seamless way) is to do something like IDM does by downloading the `<head>` metadata as necessary from `brand.learnersguild.org`.

Check out `npm run icons:fetch` on IDM.
### Tasks

- [x] use standard favicon
- [x] ensure colors match style guide

### PR DESCRIPTION
**Pull request by tannerwelsh**
Fixes #76 

- Adds colors from https://www.npmjs.com/package/@learnersguild/styles
- Uses icons from http://brand.learnersguild.org/

### CODE REVIEW

**Comment by deadlyicon** [comment]
I think you forgot to checkin your `package-lock.json` changes. Make sure you're running `node v7.5.0` and `npm 5.1.0`

File: web-server/views/layout.jade [hunk 1]
```diff
@@ -7,19 +7,16 @@ html(lang='en')
...
-    link(rel='apple-touch-icon', sizes='152x152', href='/apple-touch-icon.png')
-    link(rel="icon", href="/favicon.ico?v=2")
-    link(rel='icon', type='image/png', sizes='32x32', href='/favicon-32x32.png')
-    link(rel='icon', type='image/png', sizes='16x16', href='/favicon-16x16.png')
+    include icons_metadata
     link(rel='mask-icon', href='/safari-pinned-tab.svg', color='#5bbad5')
     meta(name='apple-mobile-web-app-title', content='curriculum')
     meta(name='application-name', content='curriculum')
-    meta(name='theme-color', content='#ffffff')
     link(rel='stylesheet', href='/assets/bootflat/css/bootstrap.min.css')
```
Comment (deadlyicon) [inline web-server/views/layout.jade]: Don't we want to replace bootflat? Or is that a next step for the branding project?

**Comment by deadlyicon** [comment]
@tannerwelsh I added some commits you should checkout. Also arent we removing bootstrap or at least removing it from this repo and moving it to brand or our own NPM modules?

**Comment by tannerwelsh** [comment]
Thanks @deadlyicon. I didn't remove bootflat b/c it seemed like too much work for the output, especially if we're going to start merging curriculum + echo soon. I mostly just wanted to get the colors aligned.

**Comment by deadlyicon** [comment]
sounds good! Thank you!

**Pull request merged**